src/run-scenario.jl

```diff
@@ -1,25 +1,21 @@
 export run_scenario
 
 """
-    run_scenario(input_folder)
     run_scenario(input_folder, output_folder)
```

Comment:
@abelsiqueira I fully agree. It looks a bit weird though (probably due to my ignorance in Julia), but since it is officially suggested, I think it is fine.